packages/react-core/src/components/Radio/Radio.tsx

```diff
@@ -7,15 +7,15 @@ import { getOUIAProps, OUIAProps, getDefaultOUIAId } from '../../helpers';
 export interface RadioProps
   extends Omit<React.HTMLProps<HTMLInputElement>, 'disabled' | 'label' | 'onChange' | 'type'>,
     OUIAProps {
-  /** Additional classes added to the radio wrapper.  This will be a div element if
-   * isLabelWrapped is true, otherwise this will be a label element.
+  /** Additional classes added to the radio wrapper. This will be a label element if
+   * isLabelWrapped is true, otherwise this will be a div element.
    */
   className?: string;
-  /** Additional classed added to the radio input */
+  /** Additional classed added to the radio input. */
   inputClassName?: string;
   /** Id of the radio. */
   id: string;
-  /** Flag to show if the radio label is wrapped on small screen. */
+  /** Flag to show if the radio label is wrapped on small screen. Will only apply if component prop is not specified. */
```

Comment:
Nit: feel like the first sentence could be tweaked as well, as the prop doesn't have anything to do with small screens. "Flag to indicate whether the radio wrapper element is a native label element" or something.

Reply:
+1